### ISSUE TITLE
Pixart Sigma images are garbled

### ISSUE THREAD

**Issue by TemuPaulMcCartney** [city96/ComfyUI_ExtraModels#57]
For some reason, Pixart Sigma images are coming out as some sort of incomprehensible mess. I followed the directions as best I could and used the sample workflow, but this is the result:
![ComfyUI_PixArt_00002_](https://github.com/city96/ComfyUI_ExtraModels/assets/171268008/499d6f76-a959-40d0-8050-b77eead83c19)

Mind you, the prompt is unchanged: "pixelart drawing of a tank with a (blue:0.8) camo pattern". I assumed this was a VAE issue, but changing the VAE has little to no effect. Everything has been uninstalled and reinstalled, but I've had no success. Any help would be sincerely appreciated.

**Comment by city96** [comment]
Yeah, this definitely seems odd, I can't really think of anything that would cause it. Could you maybe tell me the following:
- What model file(s) you used
- Python and pytorch versions
- GPU make/model

Plus maybe a copy of the workflow to be sure (you can export it as json and drag-and-drop it here)

**Comment by TemuPaulMcCartney** [comment]
Hi! Thank you so much for responding!!! Sorry for the lack of info.

**Models:**
- [PixArt-XL-2-1024-MS.pth](https://huggingface.co/PixArt-alpha/PixArt-Sigma/blob/main/PixArt-Sigma-XL-2-1024-MS.pth)
- config.json
- model.safetensors.index.json
- model-00001-of-00002.safetensors
- model-00002-of-00002.safetensors (all from [this link](https://huggingface.co/PixArt-alpha/pixart_sigma_sdxlvae_T5_diffusers/tree/main/text_encoder))
- pixart_sigma_vae.safetensors (renamed, [same file as this](https://huggingface.co/PixArt-alpha/pixart_sigma_sdxlvae_T5_diffusers/tree/main/vae))

**Python and Pytorch:**
- Python version: 3.10.11
- pytorch version: 2.3.0+cu121

**Hardware:**
- RTX 4090. Drivers are the most recent update.

Here's the workflow file! Thank you so much for your time.
[pixartbad(2).json](https://github.com/user-attachments/files/15523132/pixartbad.2.json)

**Comment by city96** [comment]
Well, no issues with the workflow or the pixart model file, so it must be something else. I assume there's no errors/warnings in the console? (you can ignore the "Not using xformers!" one, that one doesn't apply anymore).

I haven't used those exact text encoder files, but I don't see why they wouldn't work. You could test using the FP16 T5 model files from [here](https://huggingface.co/theunlikely/t5-v1_1-xxl-fp16/tree/main) I guess. Could also try setting the device to "cpu" or leaving it on "gpu" but setting the dtype to "FP16" and see if that changes anything.

![image](https://github.com/city96/ComfyUI_ExtraModels/assets/125218114/4547300a-3db7-4671-8534-30b7a3d22616)
